css/style.css

```diff
@@ -72,7 +72,7 @@ html, body {
 
 .navbar__logo {
   margin-left: 32px;
-  vertical-align: middle;
+  margin-top: 8px;
```

Comment:
on my chrome replacing vertical-align: middle; by a margin-top makes the id="logo-container" 78px height instead of 64px which is the header height. Is there a real difference in alignement ?